jsonnet/jsonnetfile.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is the filename as per the standard or can it be dependency-jsonnet.json?

Reply:
It is the auto-generated file when we run `jb init`.